core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -479,6 +494,10 @@ public SELF withRemoveImages(RemoveImages removeImages) {
         return self();
     }
 
+    public Optional<ContainerState> getServiceContainerByName(String serviceName) {
```

Comment:
I wonder if we should change the name to something like `getContainerByServiceName`...

@rnorth @kiview WDYT?

Reply:
I think that makes sense, yeah.

Reply:
@artjomka could you please adjust the name of the method? :)